docs/howto/build-run.md

```diff
@@ -222,6 +222,73 @@ version number starts with "1.8", and with JDK 11, it starts with
 [jdk-lts-roadmap]: https://adoptopenjdk.net/support.html#roadmap
 
 
+### Build failure: java.lang.UnsupportedClassVersionError, "Unsupported class file major version 57"
+
+When trying to build the Android app, you may see this error:
+
+```
+* Where:
+Build file '/Users/jappleseed/dev/zulip-mobile/node_modules/<package path>/android/build.gradle'
```

Comment:
I'll revise this soon, as I expect your comment at https://github.com/zulip/zulip-mobile/pull/4008#discussion_r401288426 applies here too.